packages/mookme/src/loaders/hooks-resolver.ts

```diff
@@ -236,13 +242,26 @@ export class HooksResolver {
       .filter((arg) => arg !== '')
       .join(' ');
     debug(`{args} will become ${args}`);
+
     for (const hook of hooks) {
       for (const step of hook.steps) {
         if (step.command.includes('{args}')) {
           debug(`matched {args} for step ${hook.name} -> ${step.name}`);
           const oldCommand = step.command;
           step.command = step.command.replace('{args}', `"${args}"`);
-          debug(`${oldCommand} -> ${step.command}`);
+          debug(`"${oldCommand}" -> "${step.command}"`);
+        }
+        if (step.command.includes('{matchedFiles}')) {
+          debug(`matched {matchedFiles} for step ${hook.name} -> ${step.name}`);
+          const oldCommand = step.command;
+          step.command = step.command.replace('{matchedFiles}', step.matchedFiles.join(' '));
+          debug(`"${oldCommand}" -> "${step.command}"`);
+        }
+        if (step.command.includes('{packageFiles}')) {
+          debug(`matched {packageFiles} for step ${hook.name} -> ${step.name}`);
+          const oldCommand = step.command;
+          step.command = step.command.replace('{matchedFiles}', hook.matchedFiles.join(','));
```

Comment:
This is just wrong and should be `stagedFiles` and/or `packageFiles` if you do not agree with other two comments.  not sure what `hook.stagedFiles` should become, It should join with a space as well to make matched block above match this one

Reply:
You're definitely right, but I think `packageFiles` is a better naming, because matched files can be defined in a completely different context than the git-related one after #68.

So the real meaning of these variables is more like so:

- `matchedFiles`: The files that are triggering this very step (including after an eventual `onlyOn`. It will match the `matchedFiles` property of the step object.
- `packageFiles`: The files that are triggering this package. It will match the `matchedFiles` property of the package hook object. 

Yet, I don't think that including the staged files list in the context of the step is a good thing, because every path will be broken (relative paths are from the package in the list of files we would provide, while the list of staged files is from the root repository).

I might be missing a use case, but from my perspective it is not a good practice to have step with impacts outside of the package's folder, other than the ones defined in the global `.hooks` folder (where `packageFiles` = `stagedFiles`).

Reply:
I am fine with packageFiles